WeScan/Edit/EditScanViewController.swift

```diff
@@ -157,7 +157,24 @@ final class EditScanViewController: UIViewController {
         }
         
         let finalImage = fixImageRotation(image: uiImage)
-        let results = ImageScannerResults(originalImage: image, scannedImage: finalImage, detectedRectangle: scaledQuad)
+        
+        /// Prepare the enhanced image using our AdaptiveThresholdCIFilter.
+        let filter = AdaptiveThresholdCIFilter()
+        filter.inputImage = filteredImage
```

Comment:
Why are we using `filteredImage` instead of `finalImage` here?

Reply:
filteredImage is a CIImage, finalImage is a UIImage.

Reply:
Isn't the `ciImage` property on it populated though? Since it's created from a CIImage.

Reply:
Fair enough. I’ll fix this and other comments tomorrow 👍

Reply:
Fixed in #ad8c234